package.json

```diff
@@ -5,7 +5,8 @@
   "scripts": {
     "start": "node ./bin/www",
     "dev": "nodemon ./bin/www",
-    "test": "jest --forceExit"
+    "test": "jest --forceExit",
+    "client:build": "cd client; yarn build"
```

Comment:
Can you make this just 'build' instead of 'client:build', because of some host that automatically runs yarn build at compilation time e.g Heroku?

Reply:
Also, this is not typescript version so, I think we don't have to build server and client

Reply:
okay

Reply:
> Also, this is not typescript version so, I think we don't have to build server and client

yes, but we need to build the client so that vue create a dist folder where we serve the pages.

Reply:
Made the adjustment.

Reply:
Thanks Bro